db/people.yml

```diff
@@ -1541,16 +1541,13 @@ jumana:
     - J.C. Umana
     - Jaylyn Umana
     - J. Umana
-  avatar: "https://s.gravatar.com/avatar/8ed1218c1874692a7474b58ed57d35b4?s=80"
-  bio: Jaylyn C. Umana grew up on Long Island, NY and is currently an undergraduate student 
-  and researcher at Columbia University pursuing a degrees in both physics and mathematics. His primary research 
-  interests are in condensed matter with a focus on transition metal oxides. He has experience with
-  synthesis as well as first-principles calculations via density functional theory (DFT).
-  education:
-      - begin_year: 2019
-      end_year: 2023
-      degree: BA
-      institution: columbiau
+  avatar: https://github.com/\x16jc-umana.png
```

Comment:
there is a typo in the name I think

Reply:
Is it because the name field doesn't match any item in the aka field?

Reply:
there is a rogue ``\x16`` that crept in somehow.  Just delete it.